project/Release.scala

```diff
@@ -19,12 +16,12 @@ object Release {
 
   lazy val releaseCommands: Setting[Seq[Command]] = commands ++= Seq(
     // Called by sbt-ci-release
-    Command.command(stryker4sPublishSigned)(
```

Comment:
I think the release process is broken by this PR. Maybe an interim solution can be to add a todo so we know it has to be fixed after merging this PR?